CHANGELOG.md

```diff
@@ -1,7 +1,12 @@
-## Changelog (Current version: 1.2.1)
+## Changelog (Current version: 1.2.2)
 
 -----------------
 
+### 1.2.2 (2021 Oct 04)
```

Comment:
We are changing the default behavior of the step here, so let's call this version 2.0.0. The legacy build system was the default, so it's a breaking change for those who used the default step inputs.